extra/format-22-influxdb-prep-data.js

```diff
@@ -63,7 +63,7 @@ fields.message_raw = toHexBuffer(msg.payload_raw);
 
 // array of names of items in msg.payload, each match to be copied as a value.
 var field_keys = [
-    "counter", "Vbat", "Vbus", "Vsys", "boot", "tempC", "tDewC", "tHeatIndexC", "p", "p0", "rh", "irradiance", "pellets", "activity"
+    "counter", "Vbat", "Vbus", "Vsys", "boot", "tempC", "tDewC", "tHeatIndexC", "p", "p0", "rh", "irradiance", "pellets", "activity", "nwTime"
```

Comment:
There's more to be done here, but it doesn't affect the firmware:

If the network time is valid, then the timestamp used for inserting the record in the database should be changed from the arrival time to the time in the packet. (At present, we use the time of arrival for the insertion timestamp. But we should not do this if the time is valid.)

Reply:
i.e., set result[0].time = GPS time from packet converted to UTC. I think the earlier decoder threw away the uplink time and set to the arrival time.

Reply:
I think it's likely that the code in the Wakefield server doesn't match the starting point for this file. Please check before changing.